tests/unit/src/main/scala/tests/QuickBuild.scala

```diff
@@ -117,7 +117,9 @@ case class QuickBuild(
           s"org.scala-lang:scala-library:$scalaVersion",
           s"org.scala-lang:scala-reflect:$scalaVersion"
         )
-      } ++ libraryDependencies
```

Comment:
Fun fact, this never added anything for Scala 3, which is why some tests worked even with bad dependencies. CrossReferenceLspSuite and RenameCrossSuite were the only ones that were problematic.

Reply:
Ach! This is due to the fact that else was operating on the whole expression with the `++`

Reply:
👀 good catch ha